src/components/DocumentationTopic/DocumentationNav.vue

```diff
@@ -49,6 +49,17 @@
         :currentTopicTags="currentTopicTags"
         :references="references"
       />
+      <NavMenuItems
+        class="nav-menu-settings"
+        :previousSiblingChildren="breadcrumbCount"
+      >
+        <LanguageToggle
+          v-if="interfaceLanguage && (swiftPath || objcPath)"
+          :interfaceLanguage="interfaceLanguage"
+          :objcPath="objcPath"
+          :swiftPath="swiftPath"
+        />
+      </NavMenuItems>
```

Comment:
You might want to expose an extra slot inside, so third parties can just easily insert NavMenu items.
```suggestion
        <slot name="menu-items" />
      </NavMenuItems>
```